packages/api/src/lib/prepareParams.js

```diff
@@ -27,6 +24,61 @@ function isEmpty(obj) {
   return [Object, Array].includes((obj || {}).constructor) && !Object.entries(obj || {}).length;
 }
 
+function processFile(paramName, file) {
+  if (typeof file === 'string') {
+    // In order to support relative pathed files, we need to attempt to resolve them.
+    const resolvedFile = path.resolve(file);
+    if (!fs.existsSync(resolvedFile)) {
+      // It's less than ideal for us to handle files that don't exist like this but because `file`
+      // is a string, it might actually be the full text contents of the file and not actually a
+      // path. Maybe we should regex here and make an attempt to determine if this is a file path
+      // and return an error if it is?
+      // @todo
+      return new Promise(resolve => {
+        resolve(undefined);
+      });
+    }
```

Comment:
I don't love this but I'm also currently unsure how to handle it. On one hand we can regex `file` to see if it *looks* like a file path and isn't just a string, but since this can be a relative path one might assume you just pass in `owlbert.png` instead of `./owlbert.png` and it should just work as `path.resolve()` handles both:

```
> path.resolve('owlbert.png')
'/Users/erunion/code/readmeio/api/owlbert.png'
> path.resolve('./owlbert.png')
'/Users/erunion/code/readmeio/api/owlbert.png'
```

And currently with this resolving to `undefined` that means if you pass in `./owlbert.png` and that file doesn't exist we'll send `./owlbert.png` as whatever parameter that's part of in your API request... which isn't really ideal either.